.github/workflows/publish.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+name: publish
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    name: Publish
+    steps:
+        - uses: actions/checkout@v2
+        - uses: actions-rs/toolchain@v1
+          with:
+              toolchain: stable
+              override: true
+        - uses: katyo/publish-crates@v1
```

Comment:
What do you think of doing just https://github.com/actions-rs/meta/issues/6#issuecomment-536440911 instead?

Reply:
I just verified that `cargo publish` executed on the repo root doesn't publish in the right order but instead just fails to execute.

Reply:
Humm, I see. Since we don't depend on any complex feature, I'm wondering if it isn't worth implementing it ourselves (i.e., changing working directories and publishing in the required order) to reduce the number of actions ran from other GitHub accounts to reduce our risks in case of account compromises. What do you think?

Reply:
Hm, makes sense! I added a bash script and use it in the action. PTAL :)

Reply:
LGTM! Thank you!